bundles/persistence/org.openhab.persistence.mysql/java/org/openhab/persistence/mysql/internal/MysqlPersistenceService.java

```diff
@@ -117,7 +119,7 @@ public void activate() {
 		sqlTypes.put("CONTACTITEM", "VARCHAR(6)");
 		sqlTypes.put("DATETIMEITEM", "DATETIME");
 		sqlTypes.put("DIMMERITEM", "TINYINT");
-		sqlTypes.put("GROUPITEM", "DOUBLE");
+		sqlTypes.put("GROUPITEM", "VARCHAR(200)");
```

Comment:
No, it isn't! Both are wrong. With my other PR Type tests for groups will be introduced.